infra/gcp/ensure-prod-storage.sh

```diff
@@ -34,6 +34,12 @@ if [ $# != 0 ]; then
     exit 1
 fi
 
+# The service account name for the image promoter.
+PROMOTER_SVCACCT="k8s-infra-gcr-promoter"
+
+# Permissions to grant GCR service account
+PERMISSIONS=("objectAdmin" "legacyBucketOwner")
```

Comment:
Not sure why this was factored out here. The old code uses these same perms anyway, so it's not a necessary change.

Reply:
Are these the correct permissions:
prod: `objectAdmin, legacyBucketOwner`
test: `objectAdmin, admin`

Independent of that, its probably good to edit an array in a constant vs. copy pasting chunks of code when we want to switch permissions.

Reply:
> Are these the correct permissions:
> prod: `objectAdmin, legacyBucketOwner`
> test: `objectAdmin, admin`

Does `admin` include `objectAdmin`? I think just `objectAdmin, legacyBucketOwner` is fine for both of them.

> 
> Independent of that, its probably good to edit an array in a constant vs. copy pasting chunks of code when we want to switch permissions.

Reply:
Cool lemme go change the test one!